enviPath_python/enviPath.py

```diff
@@ -68,6 +86,13 @@ def who_am_i(self) -> User:
         return User(self.requester, **user_data)
 
     def search(self, term: str, packages: Union['Package', List['Package']]):
+        """
+        Function designed to perform a search on an enviPath session.
+
+        :param term: the term with which the search wants to be performed
+        :param packages: the packages where the search wants to be performed
+        :return: the matching objects ids
```

Comment:
return is dict of object identifiers such as "reaction" to a list of lazy initialized objects